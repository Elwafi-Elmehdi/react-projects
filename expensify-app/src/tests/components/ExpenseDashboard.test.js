import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboard";

test("Should render correctly Dashboard page", () => {
	const wrapper = shallow(<ExpenseDashboardPage />);
	expect(wrapper).toMatchSnapshot();
});
