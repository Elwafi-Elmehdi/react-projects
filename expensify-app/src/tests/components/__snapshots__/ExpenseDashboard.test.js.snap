// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly Dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpensesList) />
</div>
`;
